legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py

```diff
@@ -1111,6 +1134,10 @@ def _fee_code_asserts(business, filing_json: dict, multiple_fee_codes, expected_
          []),
         ('CP1234567', CP_SPECIAL_RESOLUTION_TEMPLATE, 'specialResolution', Business.LegalTypes.COOP.value,
          None, False, ['SPRLN', 'OTCON']),
+        ('BC1234567', CONTINUATION_OUT_FILING, 'continuationOut', Business.LegalTypes.COMP.value,
+            Business.LegalTypes.BCOMP.value, False, []),
+        ('BC1234567', CONTINUATION_OUT_FILING, 'continuationOut', Business.LegalTypes.BCOMP.value,
+            Business.LegalTypes.COMP.value, False, []),
```

Comment:
Please include tests for `Business.LegalTypes.BC_ULC_COMPANY` and `Business.LegalTypes.BC_CCC`